mixins/models.py

```diff
@@ -115,6 +117,13 @@ class URLMixin(models.Model):
         blank=True, help_text="Overrides the internal link if set"
     )
     internal_link = PageField(related_name="+", blank=True, null=True)
+    file = FilerFileField(
+        related_name="file",
```

Comment:
Ideally this should be a plural and named something like `urlmixin_files`

Reply:
Agreed. An oversight :D